test/inductor/test_triton_cpu_backend.py

```diff
@@ -0,0 +1,48 @@
+# Owner(s): ["module: inductor"]
+import os
+
+from torch._inductor import config
+from torch._inductor.test_case import run_tests
+from torch.testing._internal.inductor_utils import HAS_CPU
+from torch.utils._triton import has_triton
+
+
+try:
+    from . import test_torchinductor
+except ImportError:
+    import test_torchinductor
+
+if has_triton():
+    import triton
+
+    TRITON_HAS_CPU = "cpu" in triton.backends.backends
+else:
+    TRITON_HAS_CPU = False
+
+
+class TritonCpuTestMixin:
+    @classmethod
+    def setUpClass(cls):
+        os.environ["TRITON_CPU_BACKEND"] = "1"
```

Comment:
It's used in the triton-cpu repo: https://github.com/triton-lang/triton-cpu/blob/ed82c339e5c9b9329d1ef3b3edea0d64e515f6ce/python/triton/runtime/driver.py#L8

Reply:
hmm, I guess it would be better if inductor set the env var globally if it knows the cpp backend is getting used. we can probably do that in the wrapper code. let me change this

Reply:
ah, nope, we would have to set it before doing any triton imports for e.g. autotuning as well, which gets kind of messy. I think in the longer run we should have a way to toggle the CPU backend without using env vars, but this is probably good enough for now

Reply:
Maybe inside inductor we should check for this and error if not set correctly?  I want users to have a good experience if they try to enable this.

Reply:
Sounds good, added an assert